src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/ContainerCleaner.java

```diff
@@ -33,9 +33,10 @@ public ContainerCleaner(
     public void process(String containerName) {
         logger.info("Started deleting dispatched blobs from container {}", containerName);
 
-        final BlobContainerClient containerClient = storageClient.getBlobContainerClient(containerName);
 
         try {
+            final BlobContainerClient containerClient = storageClient.getBlobContainerClient(containerName);
+
```

Comment:
unrelated change?

Reply:
Related - otherwise exception is thrown outside due to misconfiguration in integration tests